packages/common-scripts/lib/secp256k1_blake160_multisig.js

```diff
@@ -0,0 +1,372 @@
+const {
+  configs,
+  parseAddress,
+  minimalCellCapacity,
+  createTransactionFromSkeleton,
+} = require("@ckb-lumos/helpers");
+const { LINA } = configs;
+const { core, values, utils } = require("@ckb-lumos/types");
+const { CKBHasher, ckbHash } = utils;
+const { ScriptValue } = values;
+const { normalizers, Reader } = require("ckb-js-toolkit");
+const { Set } = require("immutable");
+
+// 65 bytes zeros
+const SIGNATURE_PLACEHOLDER =
+  "0x0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000";
+
+function ensureSecp256k1Blake160Multisig(script, config) {
+  const template = config.SCRIPTS.SECP256K1_BLAKE160_MULTISIG.SCRIPT;
+  if (
+    template.code_hash !== script.code_hash ||
+    template.hash_type !== script.hash_type
+  ) {
+    throw new Error(
+      "Provided script is not SECP256K1_BLAKE160_MULTISIG script!"
+    );
+  }
+}
+
+/**
+ * secp256k1_blake160_multisig requires S, R, M, N and public key hashes
```

Comment:
For now let's put all js/ts docs in TypeScript definition file.

Reply:
Moved to `.d.ts`